packages/runtime/runtime-utils/package.json

```diff
@@ -68,6 +68,7 @@
     "@fluidframework/container-runtime-definitions": ">=2.0.0-internal.2.2.0 <2.0.0-internal.3.0.0",
     "@fluidframework/core-interfaces": ">=2.0.0-internal.2.2.0 <2.0.0-internal.3.0.0",
     "@fluidframework/datastore-definitions": ">=2.0.0-internal.2.2.0 <2.0.0-internal.3.0.0",
+    "@fluidframework/driver-utils": "^1.3.1",
```

Comment:
It currently does live in a similar function in runtime-utils. I was trying to replace convertSummaryTreeToITree with convertSummaryTreeToSnapshotITree, so that only 1 function is needed. In order to do this, I believe the function will need to be moved elsewhere

Reply:
Feels like both dependencies are potentially sketchy to me, but driver depending on runtime is the most sketchy.  I'm not familiar with the driver-utils test that has this dependency but I suspect maybe it shouldn't be prying into the snapshot so much (also, why is "driver-utils" in the loader directory?  I feel like there's probably multiple layering issues here)